src/components/SearchInput.spec.js

```diff
@@ -0,0 +1,9 @@
+import { render, screen } from "@testing-library/react";
+import SearchInput from "./SearchInput";
+
+describe("SearchInput", () => {
+  it("should get a city through a textbox", () => {
+    render(<SearchInput />);
+    expect(screen.getByRole("textbox")).toBeInTheDocument();
+  });
+});
```

Comment:
Example for an empty function for required function props.
```suggestion
const noop = () => {} // no operation
describe("SearchInput", () => {
  it("should get a city through a textbox", () => {
    render(<SearchInput setCity={noop} />);
    expect(screen.getByRole("textbox")).toBeInTheDocument();
  });
});
```